app/src/main/res/values/styles.xml

```diff
@@ -31,6 +29,11 @@
         <item name="android:textColor">@color/colorPrimary</item>
     </style>
 
+    <style name="Toolbar">
```

Comment:
According to good practices of styles naming, this should be named `Theme.HomeAssistant.Toolbar`, that would also keep it consistent with the rest of styles.

There's also a good talk from the latest Android Dev Summit about this topic if you are interested https://youtu.be/Owkf8DhAOSo

Reply:
@gmatyszczak  It is true if it is a style.
In this case it is used as a theme overlay (`android:theme="@style/Toolbar"`).

It could be 

`<style name="ThemeOverlay.HomeAssistant.Toolbar"  
       parent="@style/ThemeOverlay.MaterialComponents.Toolbar.Primary">`

Reply:
Correct, it should be ThemeOverlay in this case ;)